arviz/plots/plot_utils.py

```diff
@@ -602,6 +603,28 @@ def color_from_dim(dataarray, dim_name):
     return colors, color_mapping
 
 
+def vectorized_to_hex(c, keep_alpha=False):
+    """Convert a color (including vector of colors) to hex
+
+    Parameters
+    ----------
+    c: Matplotlib color
+
+    keep_alpha: boolean
+        to select if alpha values should be kept in the final hex values.
+
+    Returns
+    -------
+    rgba_hex : vector of hex values
+    """
+    try:
+        hex_color=to_hex(c, keep_alpha=False)
+
+    except:
```

Comment:
The error type we want to catch should be explicit. Try calling `to_hex(["k", "r"])` yourself and see what is the error risen.